core/libs/mx-graph/src/lib/services/mx-graph-geometry-provider.service.ts

```diff
@@ -62,6 +62,8 @@ export class MxGraphGeometryProviderService {
         return new mxGeometry(x, y, ExpandedEllipseShape.expandedEllipseElementWidth, ExpandedEllipseShape.expandedEllipseElementHeight);
       case this.isRoundedBorderShape(metaModelElement):
         return new mxGeometry(x, y, ExpandedRoundBorderShape.expandedRoundBorderWidth, ExpandedRoundBorderShape.expandedRoundBorderHeight);
+      case this.isMiniShape(metaModelElement):
```

Comment:
This was removed a few weeks ago. Why do you need it?

Reply:
I will check this